data/human/human missions.txt

```diff
@@ -6438,6 +6438,402 @@ mission "Care Package to South 2"
+			`	'We at Megaparsec put the greatest attention into ensuring the safety and well-being of all our event participants and guests. Over the past forty-six hours, our search teams have fully scanned the Volax, Cardax, Moktar, Bellatrix, and Menkar systems, and we are working in conjunction with the Republic and Syndicate to fully scour their territories belonging to the circuit as well. Ms. Renard's family can rest assured that if she's out there, we'll find her.'`
+			`	'Artemis Renard, daughter of terraforming magnates August and Louisa Renard, is perhaps best known for shattering the record for the Betelgeuse Classic in last year's race, crossing the finish line nearly fifteen minutes faster than the record set by Jackson Lee over twenty years ago. She was last seen flying her ship, the "Moonfox," a white and red Lionheart Flivver with race registration B-168, in the Diphda system. Anyone with information regarding her whereabouts is encouraged to contact Syndicate or Navy authorities, as well as the Megaparsec Racing Events Commission, and may be entitled to monetary reward if such information leads to the retrieval of Ms. Renard or her ship.'"`
+			`	The broadcast then ends, going to an advertisement for the Megaparsec Splinter.`
+			
+			label renard
+			`	Several minutes later, you see Mr. Renard returning from the terminal side of the spaceport, looking frustrated. "Idiots!" he shouts. "They couldn't find their legs in a dressing room! They tell me that they had 'some network and monitoring issues' and that they don't have surveillance for the entire last leg of the race. I told Artemis to never trust these Syndicate amateurs."`
+			choice
+				`	"I'm sure they're trying their best."`
+				`	"'Network and monitoring issues' sounds suspicious."`
+					goto suspicious
+				`	"Do you have a plan for where to go next?"`
+					goto plan
+			`	"Well their best isn't very good, is it! They don't notice that their number-one pilot has been missing for days? I'll get the best lawyers in the Republic to drag Megaparsec through court 'till the only thing left they'll be able to sell will be scrap!"`
+			choice
+				`	"What you said about 'network and monitoring issues' sounded suspicious."`
+				`	"So what's the plan from here?"`
+					goto plan
+			label suspicious
+			`	Mr. Renard suddenly speaks more quietly. "I thought so too. I think it might have been sabotage."`
+			`	"What do you mean?" you ask.`
+			`	"Every day that Artemis is lost, the odds on her plummet and the bets on her competitors skyrocket. I've spend enough time making money to know that there's no faster way to bruise a competitor than scandal. I think someone sabotaged the race and went after my daughter because they knew it would make headlines, and they knew they could cash in by betting on underdogs."`
```

Comment:
```suggestion
			`	"Every day that Artemis is lost, the odds on her plummet and the bets on her competitors skyrocket. I've spent enough time making money to know that there's no faster way to bruise a competitor than scandal. I think someone sabotaged the race and went after my daughter because they knew it would make headlines, and they knew they could cash in by betting on underdogs."`
```